<launch>
    <include file="$(find ur_gazebo)/launch/ur5_bringup.launch" />
    <node pkg="Gazebo_ur5_ros_Marwan" type="cartesian_motion.py" name="cartesian_motion" output="screen"/>
</launch>
